### PR TITLE
Remove retry download

### PR DESCRIPTION
**Pull request by LePetitTim**

### CODE REVIEW

**Comment by leplatrem** [comment]
Why not set default setting to 0 instead?

**Comment by LePetitTim** [comment]
Does it really happen sometime that trying to get the tiles for the first time will give a 404 and the next occurence will not ?

**Comment by leplatrem** [comment]
A 404 probably not, but I remember the server would sometimes respond a 503 if it was too busy rendering the tiles :) 

Maybe the retry should be done only `if resp.code >= 500`. In theory 4XX are clients errors, so it doesn't make sense to retry without changing the request ;)

**Comment by LePetitTim** [comment]
ok i will change this then.

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 -e git://github.com/mapbox/mbutil.git@master#egg=mbutil
 Pillow == 1.7.7
+requests == 2.21.0
```
Comment (leplatrem) [inline requirements.txt]: add this to `setup.py`

**Comment by LePetitTim** [comment]
Do you think it is a better way ?

**Comment by leplatrem** [comment]
> Do you think it is a better way ?

Yep  :) GG!

**Pull request merged**